packages/amplify-codegen/src/commands/models.js

```diff
@@ -118,6 +119,7 @@ async function generateModels(context) {
       enableDartNullSafety,
       handleListNullabilityTransparently,
       usePipelinedTransformer,
+      enableDartNonModelGeneration,
```

Comment:
checked the current preset/visitor in master branch but could not find the place where ```enableDartNonModelGeneration``` is used.

Reply:
Synced offline to clarify this comment.